app/models/osbb.rb

```diff
@@ -0,0 +1,14 @@
+class Osbb < ApplicationRecord
+  before_save :downcase_email
+  
+  validates :name, presence: true, length: { minimum: 3, maximum: 255 }
+  validates :phone, :postcode,  numericality: { only_integer: true }
```

Comment:
I think this fields need some length validatinos

Reply:
Resolve.